bluebutton-data-pipeline-app/pom.xml

```diff
@@ -135,26 +135,57 @@
 				</executions>
 			</plugin>
 			<plugin>
-				<!-- Launch Jetty instance hosting the Blue Button FHIR server prior 
```

Comment:
It seems the master branch changes may have gotten merged strangely as I don't think they should be showing up as diffs on your pull request